addons/mail/models/mail_channel.py

```diff
@@ -58,6 +58,12 @@ def _get_default_image(self):
         help="Members of those groups will automatically added as followers. "
              "Note that they will be able to manage their subscription manually "
              "if necessary.")
+    message_unread = fields.Boolean(
+        'Unread Messages', compute='_get_message_unread',
+        help="If checked, new messages require your attention.")
+    message_unread_counter = fields.Integer(
```

Comment:
move to channel_partner instead

Reply:
and try to make it a simple search count, no query, and not user dependent